prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -1,5 +1,7 @@
 import pandas as pd
 from powersimdata.utility.distance import haversine
+from scipy.optimize import curve_fit
+from scipy.stats import linregress
 
 from prereise.gather.griddata.hifld import const
```

Comment:
Not in this PR, should we import constants explicitly instead of the entire module?

Reply:
We could, but it would be pretty long.
```from prereise.gather.griddata.hifld.const import (
    foo,
    bar,
    baz,
    ...
)

Reply:
Either way, no strong preference, just checking what we did before.